recipes/cp2k/mc/config.yaml

```diff
@@ -0,0 +1,6 @@
+name: cp2k-mc
+store: /user-environment
+spack:
+  repo: https://github.com/spack/spack.git
+  commit: develop
```

Comment:
Is it possible to fix this to a specific commit, and update it to `v0.21` when that is released?

Reply:
Yes, good point.